Add files via upload

SofiaErgenova opened the pull request.

SofiaErgenova committed:
Add files via upload

DmitryChitalov commented on Home_work_11/t_1.py:
```diff
@@ -0,0 +1,72 @@
+three_mat = Matrix(2, 2).create_matrix()
+three_mat.print_matrix()
+
+print(one_mat.mul(three_mat))
```
при сложении и умножении нужно получать новый объект класса matrix

SofiaErgenova replied:
Дмитрий, здравствуйте! Я исправила код. В форму отправки домашнего задания нужно отправить тот же пул реквест? Там будут видны правки?

SofiaErgenova committed:
Update t_1.py

Исправления: при сложении и умножении матриц возвращается новый объект класса Matrix
